vars/applier.groovy

```diff
@@ -0,0 +1,28 @@
+#!/usr/bin/env groovy
+
+class ApplierInput implements Serializable{
+    String secretName = ""
+    String registryUrl = ""
+    String inventoryPath = ""
+    String requirementsPath = ""
+}
+
+def call(Map input) {
+    call(new ApplierInput(input))
+} 
+
+def call(ApplierInput input) {
```

Comment:
Could you add some sanity checking on the input parameters, see [tagImage.groovy](https://github.com/redhat-cop/pipeline-library/blob/master/vars/tagImage.groovy#L23-L25) for example.